### ISSUE TITLE
how can i record the meeting to file like python_jibri version ?

### ISSUE THREAD

**Issue by fuqiangleon** [jitsi/jibri#87]
@aaronkvanmeerten thank you very much for your work.

**Comment by aaronkvanmeerten** [comment]
The file recording UI and experience is still in progress.  The main things to note:
You will need to implement all the file storage and delivery logic and any cleanup you wish to do, Jibri only executes a custom script after being told to stop streaming/recording.  This script receives the directory where the recording happened as a parameter.  The script needs to upload the file somewhere else and then handle any kind of communication of the location of the new file to users, outside of the jitsi meet application.

To configure it:

In jitsi-meet config.js file, set the following:

recordingType: 'jibri_file'

Then also provide a script on the jibri machine, which can upload your recording somewhere and clear the recordings directory for the next session.  There is an example of how that script might work, in the scripts/ directory.

The config.json parameter for that script is called finalize_recording_script_path, and it must be set to full path of your upload script

**Comment by fuqiangleon** [comment]
Thanks for your comment @aaronkvanmeerten 
firstly , so the jibri_file function has not been completely ,right ? because we have not found the finalize_recording_script in your comment.
secondly, i have added the recordingType: 'jibri'  in config.js , but there is not any recording button in web UI,is there any mistake of my configuration：

 recordingType: 'jibri',
enableRecording: true,
hiddenDomain: 'recorder.mydomain.com',

**Comment by fuqiangleon** [comment]
ps:recorder.mydomain.com actually is my real domain name in configuration ;)

**Comment by aaronkvanmeerten** [comment]
Correct, this is still functionality that's in development.  I'll make sure the example is up to date this week.

You would set the recordingType: 'jibri_file', instead to recordingType 'jibri' in order to set up recordings.  This replaces live stream functionality at the moment.  In development also is UI changes to provide both recordings AND live streaming functionality in the same deployment so stay tuned.

**Comment by krisalay** [comment]
Hello, 
I have added "jibri_file" in the config.js file in jitsi meet. It shows the recording button in the UI, but when i click it, it pops the message "Recording not available". Can you please tell me what can be the problem?
